### DISCUSSION TITLE
Versioning: making Query Parameter scheme case insensitive

### DISCUSSION THREAD

**Discussion by Incursus** [Azure/api-management-developer-portal#1485] [General]
Hello,

Apologies if this was asked before, my search yielded no results.

Our team is introducing versions to our APIs. If we introduce API versioning with the Query Parameter scheme, how do we make the version case insensitive?

Example:

![image](https://user-images.githubusercontent.com/12257394/136920306-7cd674d2-2989-4a0e-afea-730afd3d7f3e.png)
In the example above, querying with the query string parameter 'preview' (lower case) instead of 'Preview' would result in 404 not found status code.

We tried transformation policies as described in [Azure docs](https://docs.microsoft.com/en-us/azure/api-management/api-management-transformation-policies#SetQueryStringParameter) but that didn't help.

Any other ideas or is this just the way versioning was designed? Any help or pointing in the right direction would be greatly appreciated.

**Comment by mikebudzynski** [comment]
@Incursus, this repository is specific to the developer portal only. Please, reach out to Azure support for assistance.
